test/case/infix_services/services_basic/test.py

```diff
@@ -3,7 +3,7 @@
 Services basic
 
 Verify that basic services like mDNS and LLDP can be enabled and
```

Comment:
Very minor suggestion, if you like:
"Verify that basic services, such as mDNS and LLDP, can be enabled and disabled. Operation and non-operation are confirmed using tcpdump."